### PR TITLE
Localizes date display on quotes in editor

### LINKED ISSUE #3330: Dateformat in WYSIWYG editor looks ugly
The qoute datastring looks bad (in germany)
I don't known if is a server issue .. but if find a fix for that.

In the jquery.sceditor.smf.js find
`if (typeof attrs.date !== "undefined" && attrs.date)
            {
                attr_date = attrs.date;
                sDate = '<date timestamp="' + attr_date + '">' + new Date(attrs.date * 1000) + '</date>';
            }`

and replace with
`if (typeof attrs.date !== "undefined" && attrs.date)
            {
                attr_date = attrs.date;
                tDate = new Date(attrs.date * 1000);
                sDate = '<date timestamp="' + attr_date + '">' + tDate.toUTCString() + '</date>';
            }`

Before change:
image3

After change:
image4

### PR DESCRIPTION
**Pull request by Sesquipedalian**
Instead of displaying dates in JavaScript's default date string format (e.g. `Wed Jun 14 2017 11:31:58 GMT-0600 (MDT)`), we display them in the format for the user's locale. This still isn't exactly the same as it appears it the post display, but at least it should be clearer and more easily understandable, especially to non-English speakers.

Inspired by code originally submitted by [PortaMx](https://github.com/PortaMx) in #3335

Fixes #3330

### CODE REVIEW

File: Themes/default/scripts/jquery.sceditor.smf.js [hunk 1]
```diff
@@ -647,7 +647,9 @@ $.sceditor.plugins.bbcode.bbcode.set(
 			if (typeof attrs.date !== "undefined" && attrs.date)
 			{
 				attr_date = attrs.date;
-				sDate = '<date timestamp="' + attr_date + '">' + new Date(attrs.date * 1000) + '</date>';
+				tDate = new Date(attrs.date * 1000);
```
Comment (colinschoen) [inline Themes/default/scripts/jquery.sceditor.smf.js]: I think you want `attrs.date` -> `attr_date`

**Review by colinschoen — CHANGES_REQUESTED** [review]

**Review by colinschoen — APPROVED** [review]
LGTM

File: Themes/default/scripts/jquery.sceditor.smf.js (hunk 1 above)
Comment (Sesquipedalian) [reply Themes/default/scripts/jquery.sceditor.smf.js]: I don't think it makes much difference, but okay, sure.

Comment (colinschoen) [reply Themes/default/scripts/jquery.sceditor.smf.js]: It just is inconsistent to declare a local variable and use it in one instance and not use it in another.

Comment (Sesquipedalian) [reply Themes/default/scripts/jquery.sceditor.smf.js]: Fair enough :)

**Pull request merged**